packages/react-native-sdk/README.md

```diff
@@ -136,10 +136,10 @@ Stream's video roadmap and changelog are available [here](https://github.com/Get
 - [x] Speaking while muted
 - [x] Demo app on play-store and app-store
 - [x] Transcriptions
-- [ ] Speaker management (needs docs)
-- [ ] PiP on iOS
-- [ ] Audio & Video filters
-- [ ] CPU usage improvement
+- [x] Speaker management
+- [x] PiP on iOS
+- [x] Audio & Video filters
```

Comment:
I think we should mark audio filters still as TODO, that's the noise cancellation, no support yet on RN

Reply:
Ok, I'll break it in two